src/Helpers/Assets.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Engelsystem\Helpers;
+
+class Assets
+{
+    /** @vat string */
+    protected $assetsPath;
```

Comment:
PHP 7.4 kann typisierte Attribute :tada: 
Das könnte man hier überall reintun.

```suggestion
    protected string $assetsPath;
```